node/node_newblock.go

```diff
@@ -50,25 +57,30 @@ func (node *Node) WaitForConsensusReady(readySignal chan struct{}, stopChan chan
 					threshold = FirstTimeThreshold
 					firstTime = false
 				}
-				if len(node.pendingTransactions) >= threshold {
-					utils.GetLogInstance().Debug("PROPOSING NEW BLOCK ------------------------------------------------", "blockNum", node.blockchain.CurrentBlock().NumberU64()+1, "threshold", threshold, "pendingTransactions", len(node.pendingTransactions))
+				if len(node.pendingTransactions) >= threshold || !time.Now().Before(deadline) {
+					deadline = time.Now().Add(10 * time.Second)
```

Comment:
Make this a named constant

Reply:
change similarly to WaitForConsensusReadyv2